### DISCUSSION TITLE
SimpleLogin PGP with Mozilla Thunderbird Guide

### DISCUSSION THREAD

**Discussion by software-medic** [simple-login/app#874] [Feature Requests]
Can you create a guide on how to best setup and use SimpleLogin PGP with Mozilla Thunderbird?

<img width="1184" alt="Untitled" src="https://user-images.githubusercontent.com/103153386/162333111-44f40a78-bc93-4961-b9f9-353c5ce23e36.png">
REF: https://simplelogin.io/

The SimpleLogin main web page acknowledges that the service works perfectly with PGP Tools like Thunderbird.
I think it would be helpful to show how SimpleLogoin PGP integration works with Thunderbird.

I love this SimpleLogin privacy focused service.

**Comment by ParaplegicRacehorse** [comment]
Yes, please.  And/or link to a Thunderbird add-on.

**Comment by mfld-pub** [comment]
All you need to do is create your GPG key combination in Thunderbird under Account setup -> End To End Encryption. 

https://support.mozilla.org/en-US/kb/introduction-to-e2e-encryption

Make sure you backup and remember the backup passphrase for your PRIVATE key. Then copy your PUBLIC key to clipboard.

In simplelogin you just go to MAILBOXES and you will see a field for your e-mail address where you can paste in this PUBLIC key. Save it and you are done. Now any e-mails forwarded from any alias to this mailbox will be encrypted and sent encrypted.

**Reply by software-medic** [reply]
Thank you @mfld-pub.
I figured this out not long after my initial post.
Enabling this capability via SimpleLogin & Thunderbird allowed me to prevent big tech email providers from scanning email Subject and Body.
Happy SimpleLogin Customer
